### DISCUSSION TITLE
Implement update information within Specify

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5615] [Feature Requests]
Ben: 
> Embed some sort of news feed on the Specify home page that would show announcements and news to provide updates to users.

Grant:
> Add a check between the current version of 7 and the latest tagged release. Check every week, make message dismissable and ignorable.

There needs to be some way to check if an update that is desired is available from within the software itself.

**Comment by maxpatiiuk** [comment]
Theresa:
> There should be an option to hide the notification about a new version from the users. Some collections don't want users to be aware of the new release

(don't know the reason for that)

Ben:
> Could embed discourse "News" topic on the welcome page

**Comment by maxpatiiuk** [comment]
From @tlammer:
> a pop up the first time someone logs in after an update would be good too

On the first log in after a Sp7 instance is updated, each user should receive some kind of a message.
Maybe a link to the release notes and the documentation?

Could be combined with https://github.com/specify/specify7/issues/997
